Client/src/Utils/Theme/darkTheme.js

```diff
@@ -34,6 +35,14 @@ const darkTheme = createTheme({
 		primary: { main: "#1570ef" },
 		secondary: { main: "#2D2D33" },
 		text: text,
+		gaugePercentageText: {
+			...gaugeText,
+			fontSize: "11px"			
+		},
+		gaugeDisplayText: {
+			...gaugeText,
+			fontSize: "9px"
+		},
```

Comment:
This is better than hardcoding the values, but it does limit the use case here somewhat as these values are not reusable for anything other than gauges without being symantically incorrect.

I think it would be better to have something like

`graphHeader`

`graphSubheader`

or something like that so these values can be reused for other graphs besides the gauges?

@marcelluscaio what are your thoughts on this since you've got some experience with MUI?

Reply:
@ajhollid  Please pull the latest commit, it was addressed: https://github.com/bluewave-labs/bluewave-uptime/pull/1087/commits/5e54f5c75763e49a3efb630e7baabf71a483f0c1

Reply:
> This is better than hardcoding the values, but it does limit the use case here somewhat as these values are not reusable for anything other than gauges without being symantically incorrect.
> 
> I think it would be better to have something like
> 
> `graphHeader`
> 
> `graphSubheader`
> 
> or something like that so these values can be reused for other graphs besides the gauges?
> 
> @marcelluscaio what are your thoughts on this since you've got some experience with MUI?

@ajhollid I have updated based on this, thanks! @marcelluscaio any inputs?